packages/key-did-resolver/src/secp256k1.ts

```diff
@@ -0,0 +1,27 @@
+import * as u8a from 'uint8arrays'
+
+/**
+ * Constructs the document based on the method key
+ */
+const keyToDidDoc = (pubKeyBytes: Uint8Array, fingerprint: string): any => {
+    const did = `did:key:${fingerprint}`
+    const keyId = `${did}#${fingerprint}`
+    return {
+        "@context": "https://w3id.org/did/v1",
+        id: did,
+        publicKey: [{
+            id: keyId,
+            type: 'Secp256k1VerificationKey2018',
+            controller: did,
+            publicKeyHex: u8a.toString(pubKeyBytes, "base16"),
```

Comment:
Maybe we should set both `publicKeyHex` and `publicKeyBase58` here for future compatibility?

Reply:
makes sense 👍